### PR TITLE
Discord replaces Twitch in Footer

### LINKED ISSUE #5566: Replace Twitch with Discord in footer social icons
What? Replace the Twitch icon in our footer social icons with a Discord icon linking to our Discord.

Link: https://discord.gg/s2qDtYGnAE

Why? The owner who ran Twitch streams is no longer active and Discord needs some love.

### PR DESCRIPTION
**Pull request by st0nebraker**
This closes #5566, and /community's [#18](https://github.com/sourcegraph/community/issues/18)

### Changelog
- Places Discord icon button where the Twitch was in the Footer
- Changes the `Community` Footer item name to `Discord`

### Open question
- @jdorfman , do we want community's slug to change to => `/discord` too? Or just the footer link to the community page?

### Test
1. Ensure prettier has standardized the proposed changes.
2. Ensure the Discord icon renders & links to the correct page

#### Staging link:
https://deploy-preview-5570--sourcegraph.netlify.app/

image

### CODE REVIEW

**Review by bretthayes — DISMISSED** [review]

File: src/components/Layout/Footer.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ const Footer: FunctionComponent<Props> = ({ minimal, className }) => (
                                 <Link href="/dev-tools-quiz">Dev tools quiz</Link>
                             </li>
                             <li className="nav-item">
-                                <Link href="/community">Community</Link>
+                                <Link href="/community">Discord</Link>
```
[the remark this answers is not in the document]
Comment (st0nebraker) [reply src/components/Layout/Footer.tsx:63]: It was requested on Justin's [screen shot in his comment](https://sourcegraph.slack.com/archives/C02PSJF0QLU/p1657295009578539?thread_ts=1657247463.656319&cid=C02PSJF0QLU). I agree, which is why I left the question for Justin in the open question on the PR description. I'm happy to make the link title whichever.

[the remark this answers is not in the document]
Comment (jdorfman) [reply src/components/Layout/Footer.tsx:63]: @bretthayes checking

[the remark this answers is not in the document]
Comment (jdorfman) [reply src/components/Layout/Footer.tsx:63]: @bretthayes 

> @jdorfman , do we want community's slug to change to => /discord too? Or just the footer link to the community page?

No please leave Community there. Add Discord under Community if you don't think it's overkill

[the remark this answers is not in the document]
Comment (jdorfman) [reply src/components/Layout/Footer.tsx:63]: @st0nebraker ^ apologies for any confusion

[the remark this answers is not in the document]
Comment (st0nebraker) [reply src/components/Layout/Footer.tsx]: Sure, no problem @jdorfman !

**Review by jdorfman — APPROVED** [review]

**Pull request merged**